VaticanBAV/et/BAVet120.xml

```diff
@@ -28,148 +28,247 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-  <height>224</height>
-  <width>166</width>
-  </dimensions>
-</extent>
-<collation>
-<list>
-  <item xml:id="q1" n="A"><dim unit="leaf">2</dim></item>
-<item xml:id="q2" n="1"><num value="6">፮</num><dim unit="leaf">10</dim><locus from="3r" to="12v"></locus></item>
-<item xml:id="q3" n="2"><num value="7">፯</num><dim unit="leaf">10</dim><locus from="13r" to="22v"></locus></item>
-<item xml:id="q4" n="3"><num value="8">፰</num><dim unit="leaf">10</dim><locus from="23r" to="32v"></locus></item>
-<item xml:id="q5" n="4"><num value="9">፱</num><dim unit="leaf">10</dim><locus from="33r" to="42v"></locus></item>
-<item xml:id="q6" n="5"><num value="1">፩</num><dim unit="leaf">10</dim><locus from="43r" to="52v"></locus></item>
-<item xml:id="q7" n="6"><num value="2">፪</num><dim unit="leaf">10</dim><locus from="53r" to="62v"></locus></item>
-<item xml:id="q8" n="7"><num value="3">፫</num><dim unit="leaf">10</dim><locus from="63r" to="72v"></locus></item>
-<item xml:id="q9" n="8"><num value="4">፬</num><dim unit="leaf">10</dim><locus from="73r" to="82v"></locus></item>
-<item xml:id="q10" n="9"><num value="5">፭</num><dim unit="leaf">10</dim><locus from="83r" to="92v"></locus></item>
-<note>The quires are numbered. Quires no. 6-9 (ff. 3-42) contain the text of
-  Śәna fәṭrat, quires no. 1-5 (ff. 43-92) contain ʾAmmǝstu ʾaʿǝmāda mǝśṭir and
-  blank pages</note>
-</list>
-</collation>
-</supportDesc>
-<layoutDesc><layout writtenLines="24"></layout></layoutDesc>
-</objectDesc>
-
-<handDesc>
-<handNote xml:id="h1" script="Ethiopic">
-<seg type="ink"> Black, red</seg>
-<seg type="rubrication">Titles</seg>
-<date when="1923"></date>
-</handNote>
-</handDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc> 
+                                <support>
+                                    <material key="parchment">                                
+                            </material>
+                            </support>
+                                <extent>
+                                    <measure unit="leaf">92</measure>
+                                    <measure unit="leaf" type="blank">
+                                        <locus from="90r" to="92v"/>
+                                    </measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>224</height>
+                                        <width>166</width>
+                                    </dimensions>
+                                </extent>
+                                <collation>
+                                    <note>The quires are numbered with Ethiopic numerals. Quire marks are written in the main hand in red ink on the upper inner corner of the recto of the first folio of each quire.
+                                        Quires 1–5 (<locus from="3r" to="42v"/>) are numbered 6 to 9; quires 6–9 (<locus from="43r" to="92v"/>) are numbered 1 to 5.
```

Comment:
The numbers 1-5 are five numbers (1, 2, 3, 4, 5). The numbers 6-9 are four numbers (6, 7, 8, 9). How can quires 1-5 be numbered 6-9 and vice versa?

Reply:
Two typos: "Quires 1–5" has been corrected into "Quires 1-4", and "quires 6–9" has been corrected into "quires 5-9".
Thank you for spotting this.